### PR TITLE
Copy to Clipboard Functionality

### PR DESCRIPTION
**Pull request by JGriffCS**
@BoyWithSilverWings 
I haven't heard back from you regarding issue #10 so I figured I'd go ahead and open a PR and we could communicate here regarding how you'd like it implemented. 

I created a new component to handle the copying functionality and placed it in the sidebar. For now I've wrapped it in a FieldSet/Field combo to handle the margins/padding. Initially I tried to find a solution that didn't involve creating a new element from which to select/copy, but as far as I can tell the way the monaco editor is structured makes it impossible to perform these operators directly upon it. 

For now the button itself fills the full width of the sidebar and uses the same styles as the "Load" button and "On" switches. Please let me know if you'd like changes made to the positioning, size, styles, wording, etc...

One thing I wasn't able to figure out was how to enable/disable the button based on if the editor had content. If you have any insight into how to do this, please let me know and I'll make the necessary changes. For now the button is always active which doesn't seem to cause any issues.

Copy to Clipboard

### CODE REVIEW

**Comment by BoyWithSilverWings** [comment]
Hey @JGriffCS! First of all, Thanks for the PR.
This was an issue I had put forward as I reminder for myself and hence the lack of details, sorry about that. I don't think putting the button on the sidebar really cuts it, I will add more comments on the issue itself so that it's more findable.

File: src/components/Editors/EditorActions.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+              height: 2rem
+            `}
+        />
+      </Action>
+      <Action
+        text="Clear Editor"
+        onClick={clearEditor}
+        style={actionButtonStyles}
+      >
+        X
```
Comment (BoyWithSilverWings) [inline src/components/Editors/EditorActions.js]: Can you use × instead of a normal X?

**Comment by JGriffCS** [comment]
Hey @BoyWithSilverWings 

Per our discussion in #10 I pulled in `react-tiny-fab` and integrated the editor actions.

Copy to Clipboard v2

Let me know if you have any thoughts on colors/styles or anything else. I was also wondering where you sourced your icons from? I had originally intended to use icons for both the actions, but since the `close` icon has a border it doesn't look quite right for a "clear" function. Using just an "X" is fine, but I think the buttons might look a bit nicer if the `+` and `x` were also icons. Alternatively, if you have any suggestions for icons that more closely match the functions that would be great too.

File: src/components/Editors/EditorActions.js (hunk 1 above)
Comment (JGriffCS) [reply src/components/Editors/EditorActions.js]: Great idea! Done.

**Pull request merged**